### DISCUSSION TITLE
New find icons

### DISCUSSION THREAD

**Discussion by elax46** [elax46/custom-brand-icons#614] [Show and tell]
Since the readme file is getting bigger and bigger I'm experimenting with a new[ icon viewer/icon finder](https://elax46.github.io/custom-brand-icons/)

At the moment the problem is that all the icons have the view box problem but you are already working on it so little by little we will be able to fix everything. I ask you @rchiileea  to slowly fix all the icons that have display problems. Could you create a [favicon ](https://en.wikipedia.org/wiki/Favicon)to display at the top of the browser thanks?

**Comment by rchiileea** [comment]
ill get on this, also as far as I am aware after this batch, there should be only about 18 to 20 icons to go to complete the fix (will have to check after, having being added to the other repo has helped because the icons show there tiny and in left corner) once we have finished ill go over there and check through them again.. on the fav icon do you mean a square one with our logo?

**Reply by elax46** [reply]
yes exactly but I think the one you published is fine, I'll update everything later and check

**Comment by rchiileea** [comment]
![custom-brand-icons-facv-logo-128](https://github.com/elax46/custom-brand-icons/assets/87599807/701049ef-80a5-412f-8b38-20ea6e0a50e6)
[custom-brand-icons-facv-logo-128.zip](https://github.com/elax46/custom-brand-icons/files/14944397/custom-brand-icons-facv-logo-128.zip)
@elax46

**Reply by rchiileea** [reply]
also added a .ico file too

**Comment by rchiileea** [comment]
just checked should be on 19 files to go, once they are done ill check again to make sure they are all correct.

**Comment by milandzuris** [comment]
Hi i dont see new icons after update

**Reply by elax46** [reply]
there was a release error due to a bugfix I re-released version 4.3 with one change. Wait until you will be notified by HACS
